README.md

```diff
@@ -30,3 +30,30 @@ hack/get-terraform.sh
 ```
 
 The installer will show a series of prompts for user-specific information (e.g. admin password) and use reasonable defaults for everything else. In non-interactive contexts, prompts can be bypassed by providing appropriately-named environment variables. Refer to the [user documentation](docs/user) for more information.
+
+### Connect to the cluster
+
+#### Console
+
+Shortly after the `cluster` command completes, the OpenShift console will come up at `https://${OPENSHIFT_INSTALL_CLUSTER_NAME}-api.${OPENSHIFT_INSTALL_BASE_DOMAIN}:6443/console/`.
+You may need to ignore a certificate warning if you did not configure a certificate authority known to your browser.
+Log in using the admin credentials you configured when creating the cluster.
+
+#### Kubeconfig
+
+You can also use the admin kubeconfig which `openshift-install cluster` placed under `--dir` in `auth/kubeconfig`.
+If you launched the cluster with `openshift-install --dir "${DIR}" cluster`, you can use:
+
+```sh
+export KUBECONFIG="${DIR}/auth/kubeconfig"
+```
+
+If you didn't set `--dir`, you can use `KUBECONFIG=auth/kubeconfig`.
```

Comment:
seems like informing users that `dir` defaults to `.` seems more useful than this special case.

Reply:
> ... informing users that `dir` defaults to `.` seems more useful...

219c70f -> f9f5e8d.